Automattic/woocommerce-payments
# Update card SVG when deleting payment method from settings page.

Fixes #1953 

#### Changes proposed in this Pull Request

Pretty simple, just updating the SVG shown within the payment deletion confirmation modal.

Before:
![Screen Shot 2021-06-09 at 3 33 56 PM](https://user-images.githubusercontent.com/6723003/121494220-d5adfb00-c9d8-11eb-8500-0da7fa8b6d79.png)

After:
![Screen Shot 2021-06-09 at 3 31 59 PM](https://user-images.githubusercontent.com/6723003/121494240-d8a8eb80-c9d8-11eb-9c81-b3a3c821e441.png)


#### Testing instructions

* Make sure the UPE enabled flag is set within your testing site's database.
* On the new settings page, click 'Add Payment Method' within the `Payments accepted on checkout` section and add a new payment method.
* Click the trash icon next to one of the payment methods, observe the new SVG within the confirmation modal.

-------------------
